src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -478,7 +478,10 @@ protected override SelectionState PopulateItemsCore(string currentSelection)
                 if (category != null)
                 {
                     string name = getFullName(category);
-                    Items.Add(new DynamoDropDownItem(name, categoryId));
+                    if (name != "Cameras")
```

Comment:
Understood. I realized that it was a bad idea to begin with, and rolled that back. My bad, and thanks for the explanation.